assets/themes/restart/js/render_jobad_list_filter.js

```diff
@@ -17,6 +17,7 @@ var isLunarIndexLoaded = false;
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map
 var mapIdToJob = new Map();
 var json_data_original = [];
+var template = null;
```

Comment:
Pls rename `template` to something like `detailFormatterTemplate`.
After this - LGTM

Reply:
ok